src/VisualStudio/Core/Def/Implementation/Experimentation/KeybindingResetDetector.cs

```diff
@@ -157,6 +195,12 @@ private void UpdateStateMachine()
                         // N->E or S->E. If ReSharper was just installed and is enabled, reset NeedsReset.
                         options = options.WithChangedOption(KeybindingResetOptions.NeedsReset, false);
                     }
+                    else if (currentStatus == ReSharperStatus.Suspended && _isFirstRun)
```

Comment:
As discussed offline, this will add some spurious showing of the gold bar. We should remove this case.

Reply:
Removed the first run check